### ISSUE TITLE
No suitable kernel definition found for op Add issue

### ISSUE THREAD

**Issue by mrsabhar** [Microsoft/Windows-Machine-Learning#50]
Running latest WinMLRunning but getting error:

No suitable kernel definition found for op Add (node InceptionV1/InceptionV1/Conv2d_1a_7x7/BatchNorm/FusedBatchNorm)

I converted custom inference frozen Inception model to ONNX using VS or mmdnn. I'm getting similar error when writing new Win32 WinML desktop application from scratch.

https://stackoverflow.com/questions/52372456/windows-machine-learning-no-suitable-kernel-definition-found-for-op-add

OS: 17758 insider preview
SDK : 17754 insider preview
VS: 15.8.4

WinML Runner
GPU: Intel(R) UHD Graphics 620

=================================================================
Name: tf2onnx
Author: tf2onnx
Version: 9223372036854775807
Domain:
Description:
Path: test3.onnx
Support FP16: false

Input Feature Info:
Name: input:0
Feature Kind: Float

Output Feature Info:
Name: InceptionV1/Logits/Predictions/Reshape_1:0
Feature Kind: Float

=================================================================

Loading model...[SUCCESS]
Creating session [FAILED]
No suitable kernel definition found for op Add (node InceptionV1/InceptionV1/Conv2d_1a_7x7/BatchNorm/FusedBatchNorm)

=================================================================
Name: tf2onnx
Author: tf2onnx
Version: 9223372036854775807
Domain:
Description:
Path: test3.onnx
Support FP16: false

Input Feature Info:
Name: input:0
Feature Kind: Float

Output Feature Info:
Name: InceptionV1/Logits/Predictions/Reshape_1:0
Feature Kind: Float

=================================================================

Loading model...[SUCCESS]
Creating session [FAILED]
No suitable kernel definition found for op Add (node InceptionV1/InceptionV1/Conv2d_1a_7x7/BatchNorm/FusedBatchNorm)

**Comment by mrsabhar** [comment]
Looks like it is v4 instead of v7. I did use AI converter tool from VS (0.5.0755748). Do I need to something different or try direct tf2onnx from github. 
AI converter tool is giving old ONNX model.

**Comment by ryanlai2** [comment]
Hey mrsabhar, try using the tf2onnx converter tool: https://github.com/onnx/tensorflow-onnx

**Comment by mrsabhar** [comment]
I was able to get model running with TF-onnx. There was issue on my system where old ONNX was installed and it didnt complain :). But after uninstalling ONNX and re-installing latest ONNX 1.3 everything worked. 

Thanks for help. I'll close the issue
